### PR TITLE
Milestone 8: Add Devise

### PR DESCRIPTION
**Pull request by leopiresgoss**
## Milestone 8: Add Devise

In this milestone, @qwibbler and I:

- Removed current_user method in ApplicationController
- Installed and Set up Devise
  - Can register a new user.
  - User logs in with a combination of email and password.
  - Hashed password should be stored in the database.
  - Ask for confirmation of email.
  - Can reset password.
- Modified the views in devise for registration and login to match the wireframes from the Sneak Peak and my styling.

### CODE REVIEW

File: app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```
Comment (Meltrust) [inline app/views/devise/sessions/new.html.erb]: - [x] The log-in screen does not match the wireframe requested on the requirements.  Please, kindly make this view look more like the wireframe.

Your Implementation | Wireframe
:---:|:---:
image | image

That way, we will meet this particular requirement 💪

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+</div>
+
+<%= render "devise/shared/links" %>
```
Comment (Meltrust) [inline app/views/devise/registrations/new.html.erb]: - [x] The registration screen input fields are not aligning well.  Look:

image

Please, kindly correct this issue so the registration page looks professional 💪 

- [x] OPTIONAL: I highly suggest your make the registration screen more in line with the login screen general design presented in the log-in wireframe.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @leopiresgoss,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Registration and login are working well ✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @leopiresgoss, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**